src/api/certifications/index.js

```diff
@@ -1,6 +1,337 @@
 // @flow
 import ResourceBase from '../resourceBase';
 
+type UpdateCertificationParams = {
+    description: string,
+    title: string,
+    type: string,
+    state: string
+}
+
+type BulkCertificationParams = [
```

Comment:
I think this defines `BulkCertificationParams` to be an array of one element, so if you were to pass an array two elements to `createOrganizationCertifications`, flow would generate errors.